.travis.yml

```diff
@@ -264,6 +264,11 @@ os: linux
     cache:
       directories:
         - "$HOME/.ccache"
+        - depends/built
+        - depends/sources
+        - depends/work
+        - depends/x86_64-linux-gnu
+        - depends/x86_64-unknown-linux.gnu
```

Comment:
Is there any reason for this specific set of paths? Why not also `depends/x86_64-w64-mingw32`, for example?

Reply:
Yes skimmed over it